code/game/objects/items/storage/belt.dm

```diff
@@ -245,7 +245,7 @@
 /obj/item/storage/belt/security/ComponentInitialize()
 	. = ..()
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
-	STR.max_items = 6
+	STR.max_items = 5
```

Comment:
I wish there was a code of conduct that was enforced and forbid people from sneaking in changes that are not mentioned in the PR text body/title.

Reply:
This slot was made only to fit the club. With the club gone it's no longer needed BUT fine I've added this to description.